src/modules/dna-service/dna.background.ts

```diff
@@ -248,7 +253,49 @@ export class DnaService {
     return browserStorage.set(ACTION_QUEUE_KEY, []);
   }
 
+  async registerAllWallets() {
```

Comment:
Can you add notes for why we're doing this?

Reply:
Sure! ✅